### ISSUE TITLE
College profiles - aside elems need unique labels (aria-label/labelledby)

### ISSUE THREAD

**Issue by cjg89** [UCF/Main-Site-Theme#257]
**Describe the bug**

See title.

**Steps to Reproduce**

See:
- https://github.com/UCF/Main-Site-Theme/blob/master/taxonomy-colleges.php#L110
- https://github.com/UCF/Main-Site-Theme/blob/master/taxonomy-colleges.php#L55

**Expected Behavior**

Because multiple `<aside>`s are present on the page, each should have a unique label, using either `aria-label` or `aria-labelledby`.

**Possible Fix**

The most descriptive label we can probably add is going to be the actual text within the fact details.  Consider adding unique IDs to each `.fact-details` div, using a counter to increment the IDs (e.g. `#fact-details-1`, `#fact-details-2`), and use `aria-label` to reference those IDs on the parent aside.

**Your environment:**

n/a

**Additional context**

n/a
